sysutils/tmux/Portfile

```diff
@@ -83,6 +86,9 @@ post-destroot {
     xinstall -m 0644 ${filespath}/ftdetect-tmux.vim ${destroot}${prefix}/share/vim/vimfiles/ftdetect/tmux.vim
 }
 
-notes "If you want integration with system pasteboard consider installing port tmux-pasteboard as well"
+notes {
+    On legacy systems, consider installing port tmux-pasteboard to
```

Comment:
add a line-continuation here so that MacPorts can wrap the line at the terminal width